src/commands/test_balances.rs

```diff
@@ -27,6 +27,21 @@ impl TestBalances {
 
         TestBalances(balances)
     }
+
+    pub(crate) fn get_asset_balances_multi_user(
```

Comment:
I could make this function work without `num_users`, since I can get it from `wallets.length()`.